bulkS3upload.go

```diff
@@ -69,20 +69,27 @@ var errorLines = 0
 // and then waits on a channel for file paths that should be copied into the endpoint/bucket
 func copyWorker(bucket string, url string, accessID string, secretKey string, ssl bool, files <-chan string, nodeStats chan<- CopyResult, wg *sync.WaitGroup) {
 
+	ctx := context.Background()
 	defer wg.Done()
 
-	minioClient, err := minio.New(url, accessID, secretKey, ssl)
+	minioClient, err := minio.New(url, &minio.Options{
+		Creds: credentials.NewStaticV4(accessID, secretKey, ""),
+		Secure: ssl,
+	})
 	if err != nil {
 		log.Fatalln(err)
 	}
 	count := 0
 	for filePath := range files {
+		stringArray := strings.Split(filePath,"/")
+		objectPath := stringArray[0] + "/" + stringArray[1] + "/" + stringArray[2] + "/" + stringArray[3]
 		fullPath := rootDir + filePath
-		bytes, err := minioClient.FPutObject(bucket, filePath, fullPath, minio.PutObjectOptions{})
+		uploadInfo, err := minioClient.FPutObject(ctx, bucket, objectPath, fullPath, minio.PutObjectOptions{})
```

Comment:
uploadInfo is an object with ETag and VersionID, which are sort of useless to us in this context.

Reply:
Does the compiler complain about an unused variable? You can use "_" instead of uploadinfo if it generates a compile error.

Reply:
The compiler did complain, yes.  I've assigned to _ instead, it's cleaner than pretending to use the variable.